### DISCUSSION TITLE
Wrapping table/grid possible when too big?

### DISCUSSION THREAD

**Discussion by maribox** [typst/typst#3555] [Q&A] [answered]
Is it possible to set a table to divide itself at a row line when the table becomes larger than a page? Because if I use a default table the entire table just jumps to the next page as soon as it gets too big and if it then gets larger than an entire page it just vanishes below the page border.

![image](https://github.com/typst/typst/assets/70728036/7fcfb20d-fa90-483a-8ef9-db27a064efe6)

![image](https://github.com/typst/typst/assets/70728036/c3d61e4a-c014-4f24-aaf7-9b208068a1f3)

**Comment by laurmaedje** [comment, marked as answer]
Generally, the table should do that by default, unless you wrap it in a figure. Then, you'll need to set `show figure.where(kind: table): set block(breakable: true)`.
